simplifies READMEs and links to AVS docs (#19)
aerospike/proximus-examples: README.md

# Aerospike Vector Search (Proximus)

> [!NOTE]

> > Aerospike Vector Search(AVS) is currently in Alpha and available by invitation only. Breaking changes to our client and APIs are expected. Please refer to our [documentation](https://aerospike-vector-search.netlify.app/) for more details and to request access.

Reviewer: ```suggestion
> Aerospike Vector Search (AVS) is currently in Alpha and available by invitation only. Breaking changes to our client and APIs are expected. Please refer to our [documentation](https://aerospike-vector-search.netlify.app/vector/) for more details and to request access.
```

> ## Examples

Reviewer: ```suggestion
## Sample Applications
```

> This repo contains examples apps for utilizing Aerospike Vector Search. Currently 

Reviewer: ```suggestion
This repo contains sample apps for utilizing Aerospike Vector Search. Currently, 
```

> these are all written in Python. A brief description of example is below.

Reviewer: ```suggestion
apps are written in Python. The available apps are:
```

> * [Basic Search](./basic-search/README.md) - This is a simple application used to showcase the Python client.

Reviewer: ```suggestion
* [Basic Search](./basic-search/README.md) - A simple application used to showcase the Python client.
```

> * [Quote Search](./quote-semantic-search/) - This is a semantic text search that includes a small dataset of quotes. 

Reviewer: ```suggestion
* [Quote Search](./quote-semantic-search/) - A application for semantic text search that includes a small dataset of quotes. 
```

> * [Prism Image Search](./prism-image-search/) - This is an image search demo that can be used to search jpgs. (No dataset included in this example. )

Reviewer: ```suggestion
* [Prism Image Search](./prism-image-search/) - An image search demo that can be used to search JPEG images (no sample dataset included).
```

## Prerequisites
You don't have to know Aerospike to get started, but you do need the following:

1. A Python 3.10 - 3.11 environment and familiarity with the Python programming language (see [Setup Python Virtual Environment](./prism-image-search/README.md#setup-python-virtual-environment)).
1. The URL to your private sandbox environment (this will be provided).

## Development

> To development you can run AVS using [docker](./docker/README.md) and each of the sample application

Reviewer: ```suggestion
To develop, you can run AVS using [docker](./docker/README.md), and each of the sample applications
```
